tfx/experimental/templates/taxi/e2e_tests/test_utils.py

```diff
@@ -183,6 +203,12 @@ def _copy_template(self):
     self.assertEqual(0, result.exit_code)
     self.assertIn('Copying taxi pipeline template', result.output)
 
+class KubeflowBaseEndToEndTest(BaseEndToEndTest):
```

Comment:
I'm sorry for the confusion, but we need a separate **file** because the problem occurs when we *import* `tfx.orchestration.kubeflow.test_utils`.

Reply:
created kfp_test_utils.py